<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectDuplicateFilesWidget</class>
 <widget class="QWidget" name="DetectDuplicateFilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_build">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #B0B0B0;  /* 浅灰色 */
    color: white;               /* 白色文本 */
    border-radius: 10px;        /* 圆角边框 */
    padding: 5px 15px;          /* 内边距 */
}

QPushButton:hover {
    background-color: #A0A0A0;  /* 鼠标悬停时的颜色 */
}

QPushButton:pressed {
    background-color: #909090;  /* 按下时的颜色 */
}
</string>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>890</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>显示区</string>
   </property>
   <widget class="QListWidget" name="listWidget_processed">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>40</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_pending">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>已检出存在重复的文件列表</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>重复内容详情</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在此输入待检测重复内容的文件路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在此输入待检测重复内容的文件路径&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="title">
    <string>In</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_inPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入路径</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_inPath">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
